app/controllers/api/v1/cakes_controller.rb

```diff
@@ -10,30 +10,23 @@ def show
       end
 
       def create
-        CreateCakeCommand.new(self).run(cake_params, params[:cake][:tags])
-      end
-
-      def create_cake_succeeded(cake)
-        @cake = cake
-      end
-
-      def create_cake_failed(cake)
-        @cake = cake
+        category = Category.find(cake_params[:category_id])
+        @cake = current_user.create_cake(name: cake_params[:name], category: category)
+        if @cake.save
+          PublishToTwitterJob.set(wait_until: 1.hour.from_now).perform_later(@cake)
+        end
       end
 
       def update
         @cake = current_user.creations.find(params[:id])
         current_user.tag(@cake, with: params[:cake][:tags], on: :tags)
-        if @cake.update(cake_params.reject { |key, value| key == "tags" })
-          @cake
-        else
-          @cake
-        end
+        @cake.update!(cake_params.reject { |key, value| key == "tags" })
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.